plugins/xrandr/msd-xrandr-manager.c

```diff
@@ -1817,8 +1795,8 @@ make_menu_item_for_output_title (MsdXrandrManager *manager, MateRROutputInfo *ou
         gtk_css_provider_load_from_data (provider,css, -1, NULL);
 
         gtk_style_context_add_provider (context,
-					GTK_STYLE_PROVIDER (provider),
-					GTK_STYLE_PROVIDER_PRIORITY_FALLBACK);
+                                        GTK_STYLE_PROVIDER (provider),
+                                        GTK_STYLE_PROVIDER_PRIORITY_FALLBACK);
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.